addons/spreadsheet/static/src/pivot/pivot_helpers.js

```diff
@@ -108,3 +108,19 @@ export function parseGroupField(allFields, groupFieldString) {
         aggregateOperator,
     };
 }
+
+export const pivotRegistry = new Registry();
+
+/**
+ * Get the concrete class of an pivot, based on its definition
+ *
+ * @param {import("@spreadsheet").OdooPivotDefinition} definition
+ * @returns {import("@spreadsheet").Pivot}
+ */
+export function findPivotCls(definition) {
+    const match = pivotRegistry.getAll().find((pivot) => pivot.match(definition));
+    if (!match) {
+        throw new Error(_t("No pivot found for the given definition"));
+    }
```

Comment:
I guess this part will evolve in the very near future, right ?
isn't it too early to introduce this while there isn't a "generic pivot plugin"?